nautobot/docs/apps/index.md

```diff
@@ -0,0 +1,20 @@
+# Nautobot Apps Overview
+
+To view and search the full list of Apps, head over to the [Nautobot App Ecosystem Page on networktocode.com](https://www.networktocode.com/nautobot/apps/). Below you will only find links to the **documentation** for the apps listed in the table.
+
+## The Nautobot Community Apps
+
+Find out more about what documentation is available for the various community developed Nautobot Apps by visiting the [Community Apps](nautobot-apps.md) page.
+
+## The Network To Code Apps
+
+These Nautobot Apps have their documentation hosted as a subproject of this docs site and they are built and structured according to the **Network To Code** documentation standards.
+
+| | App Name | Description |
+|-| --- | --- |
+| ![Logo](../assets/app-icons/icon-GoldenConfiguration.png) | [Nautobot Golden Configuration](https://docs.nautobot.com/projects/golden-config/en/latest/) | Automate configuration backups, perform configuration compliance, and generate intended configurations. |
+| ![Logo](../assets/app-icons/icon-DeviceOnboarding.png) `Coming soon` | Nautobot Device Onboarding | Simplify the onboarding process of a new device by allowing the user to specify a small amount of info and having the plugin populate a much larger amount of device data in Nautobot. |
+| ![Logo](../assets/app-icons/icon-SSoT.png) | [Nautobot Single Source of Truth (SSoT)](https://nautobot-plugin-ssot.readthedocs.io/en/latest/)  | This app facilitates integration and data synchronization between various "source of truth" (SoT) systems, with Nautobot acting as a central clearinghouse for data - a Single Source of Truth|
+| ![Logo](../assets/app-icons/icon-DeviceLifecycle.png) `Coming soon` | Nautobot Device Lifecycle Management | This App works by making related associations to Devices, Device Types, and Inventory Items to help provide data about the hardware end of life notices, appropriate software versions to be running on the devices, and the maintenance contracts associated with devices. |
+| ![Logo](../assets/app-icons/icon-FirewallModels.png) | [Nautobot Firewall Models](https://nautobot-plugin-firewall-models.readthedocs.io/en/latest/) | Collection of models for constructing firewall policies in Nautobot. |
+| ![Logo](../assets/app-icons/icon-ChatOps.png) `Coming soon` | Nautobot ChatOps | Adds a chatbot to Nautobot so you can easily get data from Nautobot directly from chat. The ChatOps app is a multi-platform chatbot for network operations and engineering teams. It is built to seamlessly work across Slack, MS Teams, WebEx Teams, and Mattermost, but also as a framework to help developers add more chat platforms in the future. |
```

Comment:
Would be good to have a consistent style (verb tense, phrasing, etc.) for the Description of each app. "Automate ..." and "Simplify..." versus "This app does..." versus "Collection of models..." versus "Adds a chatbot..."

Reply:
Agree, it's a hodge-podge taken from the READMEs and documentation of each app currently. Ideally each app would have a single-paragraph description (like an abstract) that we'd replicate here.